src/main/java/com/powsybl/openloadflow/AbstractAcOuterLoopConfig.java

```diff
@@ -56,11 +56,11 @@ protected static Optional<AcOuterLoop> createMonitoringVoltageOuterLoop(OpenLoad
     }
 
     protected static Optional<AcOuterLoop> createTransformerVoltageControlOuterLoop(LoadFlowParameters parameters, OpenLoadFlowParameters.TransformerVoltageControlMode controlMode,
-                                                                                    int incrementalTransformerVoltageControlOuterLoopMaxTapShift) {
+                                                                                    int incrementalTransformerVoltageControlOuterLoopMaxTapShift, Double generatorVoltageControlMinNominalVoltage) {
```

Comment:
@annetill why a Double and not a double ? Is see no interrest to support the case where the value is null (and by the wy it would crash in the current code).